examples/advanced/quantum_volume.ipynb

```diff
@@ -0,0 +1,254 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "from examples.advanced import quantum_volume\n",
+    "import numpy as np\n",
+    "from matplotlib import pyplot as plt\n",
+    "from matplotlib.ticker import FormatStrFormatter\n",
+    "\n",
+    "# Some set-up variables. Feel free to play with these.\n",
+    "num_repetitions = 10 # This is supposed to be >= 100.\n",
+    "depths = range(2, 5) # The depths and number of qubits\n",
+    "\n",
+    "# Keep track of the ideal and noisy probabilities.\n",
+    "average_ideals = []\n",
+    "average_noisies = []\n",
+    "\n",
+    "for depth in depths:\n",
+    "    num_qubits = depth # Square matrix.\n",
+    "    print(\"Running simulation with %s qubits and a depth of %s\" % (num_qubits, depth))\n",
```

Comment:
Prefer fstrings. This applies in a lot of places.

Reply:
Oh good to know, done.